src/main/java/banking/account/policy/NormalInterestCalculator.java

```diff
@@ -0,0 +1,28 @@
+package banking.account.policy;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.NavigableMap;
+import java.util.TreeMap;
+
+public class NormalInterestCalculator implements  InterestCalculator{
+
+    private final NavigableMap<BigDecimal, BigDecimal> interestRates = new TreeMap<>();
+
+    @Override
+    public BigDecimal getInterest(BigDecimal balance) {
+        calculator();
+        BigDecimal rate = (interestRates != null) ? interestRates.floorEntry(balance).getValue(): BigDecimal.ZERO;
+        return balance.multiply(rate).setScale(0, RoundingMode.DOWN);
+    }
+
+    public void calculator() {
```

Comment:
메소드명은 동사 혹은 동사+명사 짓는 것이 좋습니다.
```suggestion
    public void calculate() {
```